src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -273,7 +273,9 @@ public void testExistingTargetFileButWithoutReadAccess()
                 + System.lineSeparator(), systemOut.getLog());
             assertEquals("Unexpected system error log", "", systemErr.getLog());
         });
-        Main.main("-c", "/google_checks.xml", "/etc/sudoers");
+        final File file = temporaryFolder.newFile("testExistingTargetFileButWithoutReadAccess");
+        file.setReadable(false);
```

Comment:
My JDK returns false and the test fails.
````
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T11:41:47-05:00)
Maven home: M:\checkstyleWorkspace\checkstyle\EMBEDDED
Java version: 1.8.0_171, vendor: Oracle Corporation
Java home: D:\Program Files\JDK8\jre
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 7", version: "6.1", arch: "x86", family: "dos"
````

Reply:
It is better to open new issue